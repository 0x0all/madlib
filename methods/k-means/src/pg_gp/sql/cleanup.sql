---------------------------------------------------------------------------
-- K-means unit test : cleanup
---------------------------------------------------------------------------
DROP FUNCTION IF EXISTS madlib_kmeans_test.random_array( INTEGER, INTEGER, INTEGER, FLOAT);
DROP FUNCTION IF EXISTS madlib_kmeans_test.rnorm( FLOAT, FLOAT);
DROP FUNCTION IF EXISTS madlib_kmeans_test.gaussian_sparse_array( FLOAT[], FLOAT);
DROP FUNCTION IF EXISTS madlib_kmeans_test.min( FLOAT, FLOAT);
DROP FUNCTION IF EXISTS madlib_kmeans_test.create_test_table( TEXT, INT, INT, INT, FLOAT);

DROP TABLE IF EXISTS madlib_kmeans_test.table123;

DROP SCHEMA madlib_kmeans_test CASCADE;