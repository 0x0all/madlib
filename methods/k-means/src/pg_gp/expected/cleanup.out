DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP TABLE
psql:cleanup.sql:12: NOTICE:  drop cascades to table madlib_kmeans_test.kmeans_out_centroids_test123
psql:cleanup.sql:12: NOTICE:  drop cascades to table madlib_kmeans_test.kmeans_out_points_test123
DROP SCHEMA
